docs/user/guides/technical-docs-seo-guide.rst

```diff
@@ -125,20 +131,47 @@ For example, if you do have a video in your docs,
 make sure the rest of that page describes the content of the video.
 
 When using images, make sure to set the image alt text or set a caption on figures.
-For Sphinx, the image and figure directives support this:
 
-.. sourcecode:: rst
+.. tabs::
+
+   .. tab:: Sphinx
+
+        For Sphinx, the image and figure directives support both alt texts and captions:
 
-    .. image:: your-image.png
-        :alt: A description of this image
+        .. code-block:: rst
 
-    .. figure:: your-image.png
+            .. image:: your-image.png
+                :alt: A description of this image
 
-        A caption for this figure
+            .. figure:: your-image.png
 
+                A caption for this figure
 
-Redirects
-~~~~~~~~~
+   .. tab:: MkDocs
+
+        The Markdown syntax fundamentally prescribes an alt text for images:
+
+        .. code-block:: md
+
+           ![Image alt text](https://dummyimage.com/600x400/){ width="300" }
+
+        If you want to use figures and captions,
+        Markdown and MkDocs do not have a built-in feature.
+        Instead,
+        you can use markdown extensions such as `md-in-html`_ to allow the necessary HTML structures for including figures:
+
+        .. code-block:: md
+
+           <figure markdown>
+              ![Image alt text](https://dummyimage.com/600x400/){ width="300" }
+              <figcaption>Image caption</figcaption>
+           </figure>
+
+.. _md-in-html: https://python-markdown.github.io/extensions/md_in_html/
+
```

Comment:
I'm a little confused, can't markdown users just write raw HTML directly? What is this extension adding?

Reply:
It's adding the ability to write HTML in Markdown.. I think it would be escaped otherwise.

The reason I understand that this is probably the way that people do it: https://squidfunk.github.io/mkdocs-material/reference/images/#image-captions

(here, they even use the word "Sadly")

Reply:
Heh yea -- it seems super hacky. 🤷